tests/plugins/widget/manual/clipboardhtmldnd.html

```diff
@@ -0,0 +1,71 @@
+<div id="editor">
+	<p>Adipisicing corporis rem repellendus vel mollitia vero?</p>
+	<div data-widget="customwidget">Widget</div>
+	<p>Consectetur dolores voluptatibus illo ipsam eveniet?</p>
+	<div data-widget="customwidget">Widget</div>
+	<p>Lorem ipsum dolor sit amet.</p>
+	<div data-widget="customnestedwidget">
+		<p>Widget</p>
+		<div class="ned">
+			<div data-widget="customwidget">Nested widget</div>
+		</div>
+	</div>
+	<p>Many,</p>
+	<p>many,</p>
+	<p>many,</p>
+	<p>many,</p>
+	<p>many,</p>
+	<p>many,</p>
+	<p>many,</p>
+	<p>many,</p>
+	<p>many,</p>
+	<p>many,</p>
+	<p>many,</p>
+	<p>many,</p>
+	<p>many</p>
+	<p>lines to test scrollhack for IE.</p>
+</div>
+
+<script>
+	if ( CKEDITOR.env.ie ) {
+		bender.ignore();
+	}
```

Comment:
Actually it was copied from the test from some other d&d tests. I removed the info about scrollhack.

> maybe it's worth to add some comment with issues references, so we can enable this test on IE/Edge once the issues are fixed?

Are they even going to be fixed? Both IE and Edge are dead, so fixing it would require introducing even more hacks to our codebase…

Reply:
Probably not, but it gives at least some clue where find the reason of ignoring the test for a full description of the issue. Currently, it's just ignored. Why? Who knows :man_shrugging: